ui/src/components/StoryDrawer.tsx

```diff
@@ -114,6 +120,14 @@ export function StoryDrawer({ story, onClose }: StoryDrawerProps): JSX.Element {
           <Grid item xs={12}>
             <StoryDrawerContentText>{content}</StoryDrawerContentText>
           </Grid>
+          <Grid item xs={12}>
```

Comment:
should we do something like `{video_url && <Grid item ... </Grid>}` so we dont render it if the video_url is blank? just wanted to make sure you tested it with stories without videos since the height is 40vw so there might be a blank space at the bottom

Reply:
yup that's a good idea, I'll add it in